test/e2e-ansible/e2e_ansible_suite_test.go

```diff
@@ -168,6 +139,38 @@ var _ = AfterSuite(func() {
 	tc.Destroy()
 })
 
+const sampleMemcachedTask = `---
+# tasks file for Memcached
+- name: start memcached
+  community.kubernetes.k8s:
+    definition:
+      kind: Deployment
+      apiVersion: apps/v1
+      metadata:
+        name: '{{ ansible_operator_meta.name }}-memcached'
+        namespace: '{{ ansible_operator_meta.namespace }}'
+      spec:
+        replicas: "{{size}}"
+        selector:
+          matchLabels:
+            app: memcached
+        template:
+          metadata:
+            labels:
+              app: memcached
+          spec:
+            containers:
+            - name: memcached
+              command:
+              - memcached
+              - -m=64
+              - -o
+              - modern
+              - -v
+              image: "docker.io/memcached:1.4.36-alpine"
+              ports:
+                - containerPort: 11211`
```

Comment:
This looks nearly identical to what `memcachedWithBlackListTask` contains for the main deployment task. Is there a way to tweak things such that we just append the extra blacklist tasks to the existing sample without touching the deployment task?

If we replace this, we open ourselves to the possibility that we change the replacement substantially enough that the original sample is no longer being tested, and that would partially defeat the purpose of using the sample in the e2e test.

Reply:
Sure, it is a good suggestion. Done. 👍